### ISSUE TITLE
Hi，question about loss function

### ISSUE THREAD

**Issue by zyy341** [jzhang-cloud/keras-unet-vocal-separation#1]
I can't find the loss function in the code，could you please to talk where loss function is？
